lib/scss/inc/_variables.scss

```diff
@@ -12,16 +12,15 @@ $screen-md-max: ($screen-lg-min - 1);
 $border-color: #f0f0f0;
 $border-inverse-color: #2f2f2f;
 $muted-bg: #f9f9f9;
-$text-muted-color: #606060;
```

Comment:
Once again, could we keep `$text-muted-color: #606060;` here according to Jayshree's feedback?